cli.go

```diff
@@ -230,6 +230,7 @@ Here is an example bash command using the CloudFlare DNS provider:
 	fmt.Fprintln(w, "\tdnspod:\tDNSPOD_API_KEY")
 	fmt.Fprintln(w, "\totc:\tOTC_USER_NAME, OTC_PASSWORD, OTC_PROJECT_NAME, OTC_DOMAIN_NAME, OTC_IDENTITY_ENDPOINT")
 	fmt.Fprintln(w, "\tsakuracloud:\tSAKURACLOUD_ACCESS_TOKEN, SAKURACLOUD_ACCESS_TOKEN_SECRET")
+	fmt.Fprintln(w, "\tnetcup:\tNETCUP_CUSTOMER_NUMER, NETCUP_API_KEY, NETCUP_API_PASSWORD")
```

Comment:
```diff
+NETCUP_CUSTOMER_NUMBER
-NETCUP_CUSTOMER_NUMER
```

Reply:
Thanks for pointing that out! Will be corrected.